spot-client/src/common/app-state/modal/modal.test.js

```diff
@@ -0,0 +1,70 @@
+import { combineReducers, createStore } from 'redux';
+
+import { routeChanged } from '../route/actions';
+
+import * as actions from './actions';
+import modalReducer from './reducer';
+import * as selectors from './selectors';
+
+describe('modal state', () => {
+    let dispatch, getState, mockModal;
+
+    /**
+     * Executes validation for no modal being configured to be shown.
+     *
+     * @private
+     * @returns {void}
+     */
+    function verifyNoModal() {
+        const state = getState();
+
+        expect(selectors.getCurrentModal(state)).toEqual({
+            modal: null,
+            modalProps: null
+        });
+        expect(selectors.isAnyModalOpen(state)).toBe(false);
+        expect(selectors.isModalOpen(state, mockModal)).toBe(false);
+    }
+
+    beforeEach(() => {
+        mockModal = jest.fn();
+        ({ dispatch, getState } = createStore(combineReducers({ modal: modalReducer })));
+    });
+
+    it('by default has no modal', () => {
+        verifyNoModal();
+    });
+
+    describe('showing modals', () => {
+        it('sets the modal component to display with its props', () => {
+            const mockProps = {};
```

Comment:
how about putting something here ? just thinking an empty object is not ideal here

Reply:
Added a key-value pair into the props. I did a fixup in the last commit and rebased onto latest master while I was at it.